native/build/CMakeLists.txt

```diff
@@ -170,6 +167,7 @@ foreach(SAMPLE_FILE ${SAMPLE_FILES})
 
 	if(IOS)
 		set_target_properties(${SAMPLE_NAME} PROPERTIES XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "iPhone Developer")
+		set(CMAKE_XCODE_ATTRIBUTE_DEVELOPMENT_TEAM E8FVX7QLET)
```

Comment:
this was necessary to get it compiling because xcode requires signing for xcode 8+. idk what to do for external people because they won't have this team though